templates/documents/general_view.html

```diff
@@ -10,6 +10,24 @@
 
 <script language="JavaScript">
 
+ function doFax(e, filePath) {literal}{
+     e.preventDefault();
+     let btnClose = '{/literal}{xl t="Cancel"|escape:"html"}{literal}';
+     let title = '{/literal}{xl t="Send To"|escape:"html"}{literal}';
```

Comment:
need to do following instead (smarty is crazy and will escape the parameter before running the function, which is why also have smarty xlt/a/j functions) (note quotes were also removed):
```
     let btnClose = {/literal}{xlj t="Cancel"}{literal};
     let title = {/literal}{xlj t="Send To"}{literal}';
```